conda_forge_tick/git_utils.xsh

```diff
@@ -238,7 +245,9 @@ def push_repo(feedstock_dir, body, repo, title, head, branch,
     ljpr.update(**pr_dict)
     return ljpr
 
-
+@backoff.on_exception(backoff.expo,
+  (requests.exceptions.RequestException, requests.exceptions.Timeout),
+  max_time=60)
```

Comment:
Can this set the `max_time` from a global or an env var? (I'm imagining that this may need tuning)